custom_otello_linter/abstract_checkers/scenario_helper.py

```diff
@@ -13,3 +13,29 @@ def get_all_steps(self, class_node: ast.ClassDef) -> List:
                 or isinstance(element, ast.AsyncFunctionDef)
             )
         ]
+
+    # Метод находит узел с декоратором и списком лейблов
+    # Лейблы, указанные в параметризации, не попадут в выборку
+    def get_allure_decorator(self, scenario_node: ast.ClassDef) -> Union[ast.Call, None]:
```

Comment:
[Optional]

Ценный комментарий! Не должен ли он быть частью функции и оформлен в docstrings?

```suggestion
    def get_allure_decorator(self, scenario_node: ast.ClassDef) -> Union[ast.Call, None]:
    """
    Метод находит узел с декоратором и списком лейблов
    Лейблы, указанные в параметризации, не попадут в выборку
    """
```

Reply:
Вопрос относится и к функциям ниже